modules/src/news.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import config
+import os
+import json
+from templates.generic import *
+from templates.text import TextTemplate
+
+NEWS_API_KEY = os.environ.get('NEWS_API_KEY', config.NEWS_API_KEY)
+
+def process(input, entities):
```

Comment:
Since we are not using `entities`, we can initialize it to `None` to signify that.